refs/browser-specs.json

```diff
@@ -79,7 +90,7 @@
     "CLOSE-WATCHER": {
         "href": "https://wicg.github.io/close-watcher/",
         "title": "Close Watcher API",
-        "status": "Editor's Draft",
+        "status": "Living Standard",
```

Comment:
hmm... browser-specs has it as discontinued and replaced by html, which this doesn't represent

Reply:
The spec is correctly represented as discontinued and replaced by html in Specref through other properties a few lines below this one:

```json
        "obsoletedBy": [
            "HTML"
        ],
        "isSuperseded": true
```

The status change is just an artefact of the way browser-specs maintains its info, even for discontinued specs (it follows the redirect, and bumps into the HTML spec whose status is "Living Standard"). The `"status"` property does not mean much for discontinued specs in any case.